services/src/main/java/org/jboss/windup/web/services/DataBackup.java

```diff
@@ -0,0 +1,149 @@
+package org.jboss.windup.web.services;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.ejb.Schedule;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import org.apache.commons.io.FileUtils;
+import org.jboss.windup.web.addons.websupport.WebPathUtil;
+import org.jboss.windup.web.furnaceserviceprovider.FromFurnace;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+@Singleton
+@Startup
+public class DataBackup
+{
+    public static final String BACKUP = "backup";
+    public static final Long MAX_AGE_DAYS = 3L;
+
+    private static Logger LOG = Logger.getLogger(DataBackup.class.getName());
+
+    @Inject
+    @FromFurnace
+    private WebPathUtil webPathUtil;
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Schedule(hour = "5,11,17,23", minute = "10")
```

Comment:
You could specify interval here. `hour="23/6"` means since 23, repeat every 6 hours.